demo/src/main/java/com/example/demo/DemoApplicationEx2.java

```diff
@@ -0,0 +1,10 @@
+package com.example.demo;
+
+public class DemoApplicationEx2 {
+
+	public void doSomething() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)